libtenzir/builtins/contexts/geoip.cpp

```diff
@@ -531,9 +552,59 @@ struct v1_loader : public context_loader {
   }
 };
 
+struct v2_loader : public context_loader {
+  explicit v2_loader(record global_config)
+    : global_config_{std::move(global_config)} {
+  }
+
+  auto version() const -> int {
+    return 2;
+  }
+
+  auto load(chunk_ptr serialized) const
+    -> caf::expected<std::unique_ptr<context>> {
+    const auto* cache_dir
+      = get_if<std::string>(&global_config_, "tenzir.cache-directory");
```

Comment:
Since we know that the cache directory option is always set, we should express that clearly to the reader of the code.

```suggestion
    const auto* cache_dir
      = get_if<std::string>(&global_config_, "tenzir.cache-directory");
    TENZIR_ASSERT(cache_dir);
```